# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/decoders/icon/mac/nsIconChannelCocoa.mm
# Commit: 73a4875ecd33
# Full Hash: 73a4875ecd33cdbf3a5ad1148c399bb21867f71a
# Author: Timothy Nikkel <tnikkel@gmail.com>
# Date: 2019-06-06 21:52:09
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1556076. Restore checking the return value of MakeInputStream in nsIconChannel on mac. r=aosmond
#   
#   Bug 1520868 accidentally removed this check.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D33892
# ==============================================================================

diff -r 325b4a93e95a -r 73a4875ecd33 image/decoders/icon/mac/nsIconChannelCocoa.mm
--- a/image/decoders/icon/mac/nsIconChannelCocoa.mm	Thu Jun 06 05:31:16 2019 +0000
+++ b/image/decoders/icon/mac/nsIconChannelCocoa.mm	Thu Jun 06 05:45:20 2019 +0000
@@ -186,7 +186,7 @@
       "security flags in loadInfo but doContentSecurityCheck() not called");
 
   nsCOMPtr<nsIInputStream> inStream;
-  MakeInputStream(getter_AddRefs(inStream), true);
+  rv = MakeInputStream(getter_AddRefs(inStream), true);
   if (NS_FAILED(rv)) {
     mCallbacks = nullptr;
     return rv;
